openreview/tools.py

```diff
@@ -440,4 +440,53 @@ def add_assignment(user, parent_group, unassigned_individual_groups, individual_
 def timestamp_GMT(year, month, day, hour=0, minute=0, second=0):
     return int((datetime.datetime(year, month, day, hour, minute, second) - datetime.datetime(1970, 1, 1)).total_seconds() * 1000)
 
+''' Create paper group, authors group, reviewers group, review non-readers group
```

Comment:
comments like these are called "docstrings", and they should be inside the function body